src/smc-webapp/project_actions.ts

```diff
@@ -352,7 +352,7 @@ export class ProjectActions extends Actions<ProjectStoreState> {
   //            or a file_redux_name
   // Pushes to browser history
   // Updates the URL
-  set_active_tab(key: string): void {
+  set_active_tab(key: string, opts: {update_file_listing: boolean} = {update_file_listing: true}): void {
```

Comment:
100% of other users of opts in CoCalc have opts as the *only option* to the function.  Don't missing positional args and opts.  Use one or the other.    This is also a weird use of opts, where this is only one option?